### DISCUSSION TITLE
Will the version 0.10 also land on Ubuntu 24.04?

### DISCUSSION THREAD

**Discussion by Chaosflo** [neutrinolabs/xrdp#3339] [Q&A] [answered]
Hey,
As the name implies, I cannot find any information when the new version 0.10 will land on Ubuntu 24.04
Currently I can only download 0.9.24 which is quite old in comparison.

Thanks =)

**Comment by metalefty** [comment, marked as answer]
Sorry, we xrdp dev team are not responsible for that. Ask Ubuntu team. 

Alternatively, some of our developers provide paid support for xrdp. xrdp v0.10 for Ubuntu 24.04 can be provided under the paid support program.

**Reply by Chaosflo** [reply]
Any links where I can ask them?

**Reply by metalefty** [reply]
I think it's here: https://discourse.ubuntu.com/

If you interested in paid support, email me to meta _at _ vmeta.jp.

**Reply by Chaosflo** [reply]
The only thing I want is that the crashes are finally stopping, that's basically all.

Its very very laborious to get this started =/
But yeah, thanks for support here!

**Comment by TO-W** [comment]
Hi,

I put the Debian version in my PPA: https://launchpad.net/~t.o/+archive/ubuntu/xrdp

I have been using it on my machine for a while now and it seems to work fine.

Best regards,
Thomas

**Comment by matt335672** [comment]
Thanks @TO-W 

@Chaosflo - if you've not used a PPA before, bear in mind that you're effectively allowing a third party to run programs with root access on your computer. For any PPA, make sure you're happy with its pedigree and the amount of risk you are taking before proceeding.

As a project, we do not recommend any particular PPA provider for xrdp. The safest way to obtain xrdp is to use your OS provider's repositories, or download and build the sources yourself from github.

**Reply by TO-W** [reply]
This is perfectly correct. It is always a risk to use third-party packages. (That is the reason why I created my PPA.)

To be on the safe side, it might probably be the best to create an own PPA and upload the Debian package https://packages.debian.org/trixie/xrdp to it. (This is what I did.)

Best regards,
Thomas
